hooks/post_gen_project.sh

```diff
@@ -1,47 +0,0 @@
-#!/usr/bin/env bash
-
-if ! make venv; then
-    echo ""
-    echo -e "\e[1m\e[31mCannot create the virtual environment.\e[0m"
-    echo ""
-    echo -e "\e[1mIs python version {{cookiecutter.python_version}} installed?\e[0m"
-    echo ""
-    echo -e "\033[1m\033[4mNext Steps\033[0m"
-    echo -e "Install python version {{cookiecutter.python_version}}."
-    echo -e "Run cookiecutter again."
-    echo ""
-    exit 1
-else
-    source venv/bin/activate && make install && make build
-fi
-
-cat << "PYTHON"
-                 .-------------------------.
-                 |         .:::::.         |
-                 |        :: ::::::        |
-                 |        ````:::::        |
-                 |  .:::::::::::::: iiii.  |
-                 | :::::::::::::::: iiiiii |
-                 | :::::: ..........iiiiii |
-                 |  ':::: iiiiiiiiiiiiii'  |
-                 |        iiiii....        |
-                 |        iiiiii ii        |
-                 |         'iiiii'         |
-                 '-------------------------'
-PYTHON
-
-
-cat << "LOGO"
-                 _       _
-                | |     | |
-              __| | __ _| |__  _   _ _ __ _   _
-             / _` |/ _` | '_ \| | | | '__| | | |
-            | (_| | (_| | |_) | |_| | |  | |_| |
-             \__,_|\__,_|_.__/ \__,_|_|   \__,_|
-
-LOGO
-
-echo -e "\033[1m\033[4mNext Steps\033[0m"
```

Comment:
Same as above:  It seems like the "next step" messages are helpful to someone not familiar with the structure.  Thoughts?  (I have no problem removing the ASCII art if it seems too gaudy.)

Reply:
### Ascii art

Not gaudy so much as just not really useful. I can add back the `daburu`, but the `python` isnt useful and just clogs up the terminal for real upside.

Reply:
Let me add more precise usage docs and this change will be easier to stomach.

Reply:
No need to re-introduce the ASCII art.  Anything that's not helpful can go.